src/test/java/com/google/api/generator/engine/ast/TernaryExprTest.java

```diff
@@ -23,9 +23,6 @@
 public class TernaryExprTest {
   @Test
   public void validTernaryExpr_primitiveType() {
-    Variable variable = Variable.builder().setName("x").setType(TypeNode.INT).build();
-    VariableExpr variableExpr = VariableExpr.builder().setVariable(variable).build();
-
     Variable conditionVariable =
         Variable.builder().setName("condition").setType(TypeNode.BOOLEAN).build();
```

Comment:
unused variableExpr, same below. The related ternary fix is separated to another PR: #187

Reply:
This PR will be hold until #187 is merged.